### ISSUE TITLE
Regression in timeToFirstFrameRasterizedMicros flutter_gallery_ios32__start_upsmoke_catalina_start_up_ios

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#68718]
See https://flutter-flutter-perf.skia.org/e/?begin=1603144085&end=1603212591&keys=Xe58c4638b290dfd947817edd92e937ac&num_commits=50&request_type=1&xbaroffset=22096.

First occurred at https://github.com/flutter/flutter/pull/68538, so I'm wondering if this caused by the test and not really the regression. @jonahwilliams can you confirm?

**Comment by jonahwilliams** [comment]
All of the startup tests now uninstall after the test run to avoid filling up temp during re-install. This might change the benchmark numbers. I also decreased the iterations to 5.


FYI @liyuqian

**Comment by liyuqian** [comment]
I just created https://github.com/flutter/flutter/issues/68728 as the uninstall seem to have a wide impact on perf metrics. See also https://github.com/flutter/flutter/issues/68367 as a related issue.

**Comment by kf6gpe** [comment]
[perf triage] This may have recovered --- see https://flutter-flutter-perf.skia.org/e/?begin=1605143043&end=1605551441&queries=sub_result%3DtimeToFirstFrameRasterizedMicros&requestType=0.

**Comment by kf6gpe** [comment]
Nope; I misspoke. See https://flutter-flutter-perf.skia.org/e/?begin=1602715025&end=1605411482&queries=test%3Dsmoke_catalina_start_up%26test%3Dsmoke_catalina_start_up_ios&requestType=0.

**Comment by matanlurey** [comment]
It's been 5 years.
